src/render/blog/release-1_5_0.html.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Knot.x 1.5.0 released!
+description: We have just released a minor release Knot.x 1.5.0.
+author: skejven
+keywords: release
+order: 1
+date: 2018-12-19
+knotxVersions:
+    - 1.5.0
+---
+# Knot.x 1.5.0
+
+## Release Notes
+
+### [knotx-core](https://github.com/Cognifide/knotx)
+ - [PR-468](https://github.com/Cognifide/knotx/pull/468) - Fragment processing failure handling (configurable fallback)
+ - [PR-465](https://github.com/Cognifide/knotx/pull/465) - Action Knot functionality moved to [Knot.x Forms](https://github.com/Knotx/knotx-forms).
+ - [PR-467](https://github.com/Cognifide/knotx/pull/467) - fixed typo in logger format (URI already contains leading slash)
+ - [PR-473](https://github.com/Cognifide/knotx/pull/473) - mark Handlebars Knot as deprecated on behalf of [Knot.x Template Engine](https://github.com/Knotx/knotx-template-engine).
+
+### [knotx-dependencies](https://github.com/Knotx/knotx-dependencies)
+- [PR-16](https://github.com/Knotx/knotx-dependencies/pull/16) - Removed `knotx-adapter-common` dependency
+- [PR-15](https://github.com/Knotx/knotx-dependencies/pull/15) - Configure Knot.x Template Engine in BOM file
+- [PR-12](https://github.com/Knotx/knotx-dependencies/pull/12) - Configure Knot.x Forms in BOM file
+
+### [knotx-junit5](https://github.com/Knotx/knotx-junit5)
+- [PR-5](https://github.com/Knotx/knotx-junit5) - Html Markup Assertion to test html body content
+
+### [knotx-forms](https://github.com/Knotx/knotx-forms)
+First release of the Knot.x Forms module that replaces [Knot.x Action Knot](https://github.com/Cognifide/knotx/wiki/ActionKnot)
+(which is now deprecated and will be removed soon).
+Read more in the *Upgrade notes* section.
+
+### [knotx-data-bridge](https://github.com/Knotx/knotx-data-bridge)
+First release of the [Knot.x Data Bridge](https://github.com/Knotx/knotx-data-bridge) module
+that replaces old [Knot.x Service Knot](https://github.com/Cognifide/knotx/wiki/ServiceKnot)
+(which is now deprecated and will be removed soon).
+
+### [knotx-template-engine](https://github.com/Knotx/knotx-template-engine)
+- [PR-37](https://github.com/Knotx/knotx-data-bridge/pull/37) - implementation of [fallback handling](https://github.com/Cognifide/knotx/issues/466) in data bridge
```

Comment:
This also ;)

Reply:
Yeah, looks like something gone wrong...
Thanks for spotting, fixed.